src/xiopak.cpp

```diff
@@ -601,17 +600,22 @@ int xfExist(	// determine file existence
 	// return to caller or use tmpstr
 	const char* tPath = strTrimEX( fPathChecked, strTrimB( fPath), "\\");
 	if (!IsBlank( tPath))
-	{	struct _stat stat;
-		int statRet = _stat( tPath, &stat);
-		if (statRet == 0)		// if file found
-			ret = (stat.st_mode & _S_IFDIR)  ? 3
-			    : (stat.st_mode & _S_IFREG)  ? 1 + int( stat.st_size > 0)
-				:                              0;	// other (e.g. volume "C:")
-		else if (statRet==-1 && errno == ENOENT)
-			ret = 0;	// clean not found
-		else
-			ret = -1;	// unknown error
+	{
```

Comment:
This all looks reasonable, but it's very time-consuming (i.e. doubles the work done) if the reviewer has to verify that all the new code performs the same logic as the old code. Isn't there a place where we can put tests for these functions now?

Reply:
I added a new test for non-found file logic.

Reply:
Did you commit `no_file.cse`?

Reply:
This was solved in slack. No `no_file.cse` is need it.